### PR TITLE
enemy healthbar

### PR DESCRIPTION
**Pull request by lewisyng**
added colored healthbar to enemy instead of counter

### CODE REVIEW

File: enemies.js [hunk 1]
```diff
@@ -4,6 +4,14 @@ const enemySprites = [],
   deadEnemies = [];
 let spawnedEnemies = 0;
 
+const healthColor = {
+  100: "green",
```
Comment (Rehaan99) [inline enemies.js]: This solution won't work as there are some towers that deal more than 20 damage and there will be some instances of enemies having more than (and less than) 100 health, we need something with a percentage range of the enemy health that defines their colour.

**Review by Rehaan99 — CHANGES_REQUESTED** [review]
See comment on PR.

File: enemies.js (hunk 1 above)
Comment (lewisyng) [reply enemies.js]: Added function that colors the healthbar based on relative health

File: enemies.js [hunk 2]
```diff
@@ -5,11 +5,25 @@ const enemySprites = [],
 let spawnedEnemies = 0;
 
 const healthColor = {
```
Comment (Rehaan99) [inline enemies.js]: Doesn't look like this is being used anywhere now, it can be deleted.

**Review by Rehaan99 — CHANGES_REQUESTED** [review]
PR looks good, just need to remove `healthColor`.

**Comment by lewisyng** [comment]
Deleted

**Review by Rehaan99 — APPROVED** [review]

**Comment by Rehaan99** [comment]
Nice work.

**Pull request merged**